Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
         choice(name: "BUILD_TYPE", choices: ["Release", "Debug"], description: "")
         choice(name: "CHANNEL", choices: ["nightly", "dev", "beta", "release", "development"], description: "")
         string(name: "SLACK_BUILDS_CHANNEL", defaultValue: "#build-downloads-bot", description: "The Slack channel to send the list of artifact download links to. Leave blank to skip sending the message.")
-        string(name: "CHROMIUM_SRC", defaultValue: "https://github.com/chromium/chromium", description: "or use https://chromium.googlesource.com/chromium/src.git")
+        string(name: "CHROMIUM_SRC", defaultValue: "https://github.com/brave-chromium/chromium-81.0.4044.83.git", description: "or use https://chromium.googlesource.com/chromium/src.git")
```

Comment:
actually I think the default value should stay on `https://github.com/chromium/chromium`, but it's never used anyway, right? It will always use https://github.com/brave/brave-browser/pull/9039/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R33 ?

Reply:
that has been taken out in a newer PR